src/Microsoft.DotNet.VersionTools/Dependencies/FileRegexUpdater.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.VersionTools.Dependencies
+{
+    public abstract class FileRegexUpdater : IDependencyUpdater
+    {
+        public string Path { get; set; }
+        public Regex Regex { get; set; }
```

Comment:
This is to support cases where we have hard-coded version strings in other files, ex. dir.props, or like in the CLI:  https://github.com/dotnet/cli/blob/rel/1.0.0/build_projects/dotnet-cli-build/CliDependencyVersions.cs#L10-L12